test_api2_source_conversation_seen is failing with:

```
>           response = app.post(
                url_for("api2.data"),
                json={"events": [asdict(event)]},
                headers=get_api_headers(journalist_api_token),
            )

...
                for uuid, item in result.items.items():
>                   response.items[uuid] = item.to_api_v2(minor) if item is not None else None
E                   AttributeError: 'str' object has no attribute 'to_api_v2'
```

--- a/securedrop/journalist_app/api2/events.py
+++ b/securedrop/journalist_app/api2/events.py
@@ -315,7 +315,7 @@ class EventHandler:
             event_id=event.id,
             status=(EventStatusCode.OK, None),
             sources={source.uuid: source},
-            items={item.uuid: item.uuid for item in seen_items},
+            items={item.uuid: item for item in seen_items},
         )
 
     @staticmethod

--- a/securedrop/tests/test_journalist_api2.py
+++ b/securedrop/tests/test_journalist_api2.py
@@ -1366,19 +1366,14 @@ def test_api2_source_conversation_seen(
         # Items within bound must be marked seen; items outside must not be
         for item_uuid, count in initial_counts.items():
             submission = Submission.query.filter(Submission.uuid == item_uuid).one_or_none()
-            reply = Reply.query.filter(Reply.uuid == item_uuid).one_or_none()
 
             if count <= upper_bound:
                 assert item_uuid in response.json["items"]
                 assert response.json["items"][item_uuid] is not None
                 if submission is not None:
                     assert submission.seen is True
-                else:
-                    assert len(reply.seen_replies) > 0
             elif submission is not None:
                 assert submission.seen is False
-            else:
-                assert len(reply.seen_replies) == 0
 
         # Resubmission must yield "Already Reported" (208)
         res2 = app.post(

